src/main/webapp/app/overview/course-score-calculation.service.ts

```diff
@@ -1,16 +1,18 @@
 import { Injectable } from '@angular/core';
 import { Result } from 'app/entities/result.model';
 import { Course } from 'app/entities/course.model';
-import { Exercise } from 'app/entities/exercise.model';
+import { Exercise, ExerciseType } from 'app/entities/exercise.model';
 import * as moment from 'moment';
 import { Moment } from 'moment';
 import { StudentParticipation } from 'app/entities/participation/student-participation.model';
-import { Participation } from 'app/entities/participation/participation.model';
+import { InitializationState, Participation } from 'app/entities/participation/participation.model';
 
 export const ABSOLUTE_SCORE = 'absoluteScore';
 export const RELATIVE_SCORE = 'relativeScore';
 export const MAX_SCORE = 'maxScore';
 export const PRESENTATION_SCORE = 'presentationScore';
+export const REACHABLE_SCORE = 'reachableScore';
```

Comment:
I guess this is not in the scope of this ticket but I am unsure why we use a Map<string,number> here and export the keys in this complicated fashion.  Would something like 
`interface CourseScores {
    absoluteScore: number;
    relativeScore: number;
}
`
not be much cleaner and more in the spirit of typescript?